docs/Tutorials/Quickstarts/Private-Network-Quickstart.md

```diff
@@ -320,112 +325,88 @@ npm install -g truffle
+    
 
-Create a `pet-shop-tutorial` directory and move into it:
-
-```bash
-mkdir pet-shop-tutorial
-
-cd pet-shop-tutorial
-```
-
-Unpack Pet Shop [Truffle box](https://truffleframework.com/boxes): 
-
-`truffle unbox pet-shop`
+The source code for the smart contract and DApp are in the `pet-shop` folder. With the blockchain running and Metamask connected to it, run the command:
 
-Install the [Truffle wallet](https://github.com/trufflesuite/truffle-hdwallet-provider):
-Note with Truffle 5, you must use a Web3 1.0 enabled wallet or the Truffle tasks will hang.
+ ```bash tab="Linux/MacOS"
+ ./run-dapp.sh
+ ```
 
-```bash
-npm install --save @truffle/hdwallet-provider
-```
+The script first compiles the contract, then performs a migration to the blockchain network and then runs some tests. 
+You can then search for the deployed contracts and transactions in the block explorer using the addresses displayed in your output.
+Output similar to the following is displayed (your addresses will differ):
 
-!!!note
-    `npm` requires `sudo` on Linux.
+```log
+Compiling your contracts...
+===========================
+> Everything is up to date, there is nothing to compile.
 
-#### Modify the Pet Shop Example
-
-Modify the `truffle-config.js` file in the `pet-shop-tutorial` directory to add our wallet provider. The following shows the
-code with placeholders to change as directed below:
-
-```javascript
-const PrivateKeyProvider = require("truffle-hdwallet-provider");
-const privateKey = "8f2a55949038a9610f50fb23b5883af3b4ecb3c3bb792cbcefbd1542c692be63";
-
-module.exports = {
-  // See <http://truffleframework.com/docs/advanced/configuration>
-  // for more about customizing your Truffle configuration!
-  networks: {
-    development: {
-      host: "127.0.0.1",
-      port: 7545,
-      network_id: "*" // Match any network id
-    },
-    quickstartWallet: {
-      provider: () => new PrivateKeyProvider(privateKey, "<YOUR HTTP RPC NODE ENDPOINT>"),
-      network_id: "*"
-    },
-  }
-};
-```
 
-Replace `<YOUR HTTP RPC NODE ENDPOINT>` with your HTTP RPC node endpoint (for example, `http://localhost:32770/jsonrpc`).
+Starting migrations...
+======================
+> Network name:    'quickstartWallet'
+> Network id:      2018
+> Block gas limit: 0xf7b760
 
-The private key is the miner address, which contains Ether. 
 
-Once this is done, follow the [Truffle tutorial steps](https://truffleframework.com/tutorials/pet-shop#directory-structure)
-up to Step 3 in the [Migration section](https://truffleframework.com/tutorials/pet-shop#migration).
+1_initial_migration.js
+======================
 
-We're using the private network instead of [Ganache](https://truffleframework.com/ganache), so skip steps 3, 4, and 5 in
-the [Migration section](https://truffleframework.com/tutorials/pet-shop#migration). 
+   Replacing 'Migrations'
+   ----------------------
+   > transaction hash:    0x10f5ebaca0e89fa84bcae19d6848dde19817ea3cf270be0fd0ab52f041a02c61
```

Comment:
All of this may be too much detail?

Reply:
You can use an example code block with two tabs, command and result so the detailed result is available for readers but will not be visible initially. See https://wiki.hyperledger.org/display/BESU/MkDocs+And+Markdown+Guide#MkDocsAndMarkdownGuide-Admonition

Reply:
Nice :) done